tests/ci/azure_pipeline_test/dsvm_notebook_win_cpu.yml

```diff
@@ -37,10 +47,6 @@ jobs:
       failTaskOnFailedTests: true
     condition: succeededOrFailed()
 
-  - script: |
-      del /q /S %LOCALAPPDATA%\Temp\*
-      for /d %%i in (%LOCALAPPDATA%\Temp\*) do @rmdir /s /q "%%i"
-    displayName: 'Remove Temp Files'
-    enabled: false
-    condition: succeededOrFailed()
-
+  - template: .ci/steps/reco_conda_clean_win.yml@aitemplates
```

Comment:
I like this idea, I don't know what the others would say. 

I think though that it would be good to explain that this file comes from the repo https://github.com/microsoft/AI/blob/master/.ci/steps/reco_conda_clean_win.yml somewhere, either in this file or in the readme

Reply:
I have updated the readme with a quick summary and links for each template.